specs/SystemCursorId

```diff
@@ -0,0 +1,87 @@
+# Background
+
+In visual hosting, the current cursor image cannot be determined easily by the
```

Comment:
In our API and docs we should use consistent name here. We don't use visual in our API surface we have composition hosting using the composition controller.

Reply:
In my 2nd commit, I've corrected this but I noticed our docs for CoreWebView2CompositionController uses visual hosting (Perhaps we should change this as well):
    // Summary:
    //     This class is an extension of the CoreWebView2Controller class to support visual
    //     hosting.
    public class CoreWebView2CompositionController